multi_cgra/test/MeshMultiCgraTemplateRTL_test.py

```diff
@@ -152,7 +152,10 @@ def run_sim(test_harness, max_cycles = 200):
   test_harness.sim_tick()
 
 
-def test_mesh_multi_cgra_universal(cmdline_opts, multiCgraParam = None):
+def test_mesh_multi_cgra_universal(cmdline_opts):
```

Comment:
I think your current param looks good: https://github.com/tancheng/VectorCGRA/blob/04b45159f4b69b840d302f31e453d33381c577e9/multi_cgra/test/MeshMultiCgraTemplateRTLScale_test.py#L10

i.e.,
```
def test_mesh_multi_cgra_universal(cmdline_opts, multiCgraParam = None):

if multiCgraParam == None:
  arch_file = os.path.join(os.path.dirname(__file__), "arch.yaml")
  parser = Parser(arch_file)
  multiCgraParam = parser.parse_multi_cgra_param()
```

How does this sound?